log4j-core/src/main/java/org/apache/logging/log4j/core/tools/picocli/CommandLine.java

```diff
@@ -4402,8 +4401,10 @@ public static IStyle bg(String str) {
                 public String off() { return CSI + (fgbg + 1) + "m"; }
             }
             private static class StyledSection {
-                int startIndex, length;
-                String startStyles, endStyles;
+                int startIndex;
+                final int length;
+                String startStyles;
+                final String endStyles;
```

Comment:
These all can very well be `private final`, I guess.

Reply:
Yep. changed